acir/src/circuit/opcodes.rs

```diff
@@ -224,44 +154,3 @@ impl std::fmt::Debug for Opcode {
         std::fmt::Display::fmt(self, f)
     }
 }
-
-#[test]
-fn serialization_roundtrip() {
```

Comment:
Couldn't hurt to still have the serialization_roundtrip test? At least here for the main circuit where we still have `read` and `write`

Reply:
I left it in for the main Circuit in `acir/src/circuit/mod.rs` -- The derive(Serialize, Deserialize) should take care of it for opcodes though

Reply:
Oops I missed that I thought you deleted it. Perhaps we should copy-paste some of the directives, opcodes, etc. from the other serialization tests into the test in `acir/src/circuit/mod.rs` for completeness sake

Reply:
Yep makes sense